.github/workflows/lint.yml

```diff
@@ -0,0 +1,17 @@
+name: golangci
```

Comment:
Rename workflow to "lint". I'll rename the "Main" workflow to "test" for coherence.